src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -90,14 +100,23 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
       <div className={styles.viewInLinks}>View in GB</div>
 
       <div className={styles.geneViewTabs}>
-        These are the Entity Viewer tabs...
+        <GeneViewTabs />
       </div>
-      <div className={styles.transcriptsListWrapper}>
-        {basePairsRulerTicks && (
-          <DefaultTranscriptslist
-            gene={props.gene}
-            rulerTicks={basePairsRulerTicks}
-          />
+      <div className={styles.geneViewTabbedContent}>
```

Comment:
Same as above: `tabbed` -> `tab`

Reply:
Related to the previous comment. That's done